src/components/CreateTargetForm/index.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { func, shape, number, string, oneOfType, bool } from 'prop-types';
+import { View, Text, TouchableOpacity, Platform } from 'react-native';
+import { LOADING } from '@rootstrap/redux-tools';
+import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view';
+
+import ErrorView from 'components/common/ErrorView';
+import Button from 'components/common/Button';
+import Input from 'components/common/Input';
+import strings from 'locale';
+import useForm from 'hooks/useForm';
+import useValidation from 'hooks/useValidation';
+import createTargetValidations from 'validations/createTargetValidations';
+import useCreateTargetForm from 'hooks/useCreateTargetForm';
+import useTextInputProps from 'hooks/useTextInputProps';
+import { BLACK } from 'constants/colors';
+import common from 'constants/commonStyles';
+import fonts from 'constants/fonts';
+import styles from './styles';
+
+const FIELDS = {
+  areaLength: 'areaLength',
+  title: 'title',
+};
+const { TARGET } = strings;
```

Comment:
you can keep destructuring TARGET so you don't repeat TARGET all the time, like this: 

`const { TARGET: {topicsError, loadingTopics, topicPlaceholder,... }} = strings`

Also, since you still need access to hole TARGET to pass it to useCreateTargetForm, you could do it like this: 

`const { TARGET,  TARGET: {topicsError, loadingTopics, topicPlaceholder,... }} = strings`

Maybe in this case it is better to repeat TARGET since otherwise, you will get very long destructuring, but I mention it in case you didn't know.